2022/typescript/day11.ts

```
@@ -1,63 +1,64 @@
 
-    divisorsMultiplied *= Number(divisor);
-    const calm = (x: number) => (isWorrying ? Math.floor(x / 3) : x % divisorsMultiplied);
-    const test = (x: number) => (x % Number(divisor) === 0 ? Number(yes) : Number(no));
-    const operation = (old: number) => eval(op);
-    monkeys.push(createMonkey(startItems, operation, calm, test));
-  });
+    divisorsMultiplied *= Number(divisor)
+    const calm = (x: number) => (isWorrying ? Math.floor(x / 3) : x % divisorsMultiplied)
+    const test = (x: number) => (x % Number(divisor) === 0 ? Number(yes) : Number(no))
+    // noinspection JSUnusedLocalSymbols
+    const operation = (old: number) => eval(op)
```

Biome

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)